build/gulp/scaffold/rollup.config.js

```diff
@@ -0,0 +1,110 @@
+import replace from 'rollup-plugin-replace'
```

Comment:
this should be placed somewhere under `test-projects` at the very least (we should create corresponding dir) - so that this client-oriented tests functionality will be cohesively structured

Reply:
It's already placed in the separate dir. I don't want to be so specific for now.

Reply:
yes, but the same reasoning would apply well in case it will be put in _any_ dir - thus, I am not sure that this is a valid argument. 

The reasoning behind my ask is the following: currently, with the dir introduced at the place where you are suggesting, we would have the following project structure:

```
/build/gulp
  /scaffold
  /tasks
       /test-projects
```

So, reasonable question is why `scaffold` directory is placed at this level of folder hierarchy? Is its content shared by any agents/tasks? The answer is "no" - the only place where it is used is `test-projects` module. Thus, I would suggest to put these bits to the module those belong to at the moment.

Reply:
> Is its content shared by any agents/tasks? The answer is "no" - the only place where it is used is test-projects module.

At now, yes. But in the future, I think that we will have other tasks that will should also have files and templates.

Reply:
@kuzhelov @mnajdova you convinced me and I made these changes 👍